specification/scvmm/resource-manager/readme.md

```diff
@@ -24,17 +24,46 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 
 These are the global settings for the scvmm.
 
-```yaml
+``` yaml
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2022-05-21-preview
+tag: package-preview-2023-04
 ```
 
+
+### Tag: package-preview-2023-04
+
+These settings apply only when `--tag=package-preview-2023-04` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-04'
+input-file:
+  - Microsoft.ScVmm/preview/2023-04-01-preview/scvmm.json
+suppressions:
+  - code: RESPONSE_STATUS_CODE_NOT_IN_SPEC
+    reason: All POST actions are long running operations and never return 200 in the response.
```

Comment:
200 is an allowed status code for POST (LOR or not)

Reply:
Addressed ✅